recipe/meta.yaml

```diff
@@ -18,24 +18,23 @@ build:
 
 requirements:
   host:
-    - python >=3.4
+    - python >=3.9
     - pip
     - setuptools
   run:
-    - bw2parameters
-    - bw_processing
-    - lxml
+    - python >=3.9
+    - scipy
     - numpy
     - peewee >=3.9.4
     - platformdirs
-    - python >=3.5
-    - requests >=1.1.0
-    - scipy
-    - stats_arrays
     - tqdm
     - voluptuous
     - whoosh
     - wrapt
+    - bw2parameters
+    - bw_processing
+    - bw-projects >=2.1.0
```

Comment:
Sorry, we don't need this (bw_projects) anymore.

Reply:
...sorry, I forgot.